src/base/preferences.cpp

```diff
@@ -1241,32 +1241,32 @@ void Preferences::setRssHSplitterSizes(const QByteArray &sizes)
 
 QStringList Preferences::getRssOpenFolders() const
 {
-    return value("Rss/open_folders").toStringList();
+    return value("GUI/RSSWidget/OpenedFolders").toStringList();
```

Comment:
I had some conflicts with my new config options (because of some strange/specific/buggy QSettings behavior I found). I prefer rename old options to proper names instead of trying to fit the new names.
This will break only RSS GUI settings and make it invisible again. But it would further indicate to the user that there is now a new (revised) RSS subsystem.